apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx

```diff
@@ -65,6 +65,9 @@ export const BreakpointsSelector = ({
         type="single"
         value={selectedBreakpoint.id}
         onValueChange={(breakpointId: string) => {
+          if (!breakpointId) {
```

Comment:
according to types, it can't be empty, so something is wrong here

Reply:
Maybe we can use length?

Reply:
@TrySound please check this out, strange, it shouldn't be an empty string ever from what I know

Reply:
https://www.radix-ui.com/docs/primitives/components/toggle-group#item

You can see the toggles can be turned on and off. It sends a empty string once the toggle to the `onValueChange` function.